python/ray/_private/services.py

```diff
@@ -31,6 +31,11 @@
 # True if processes are run in the valgrind profiler.
 RUN_RAYLET_PROFILER = False
 
+# The number of seconds to wait for the Raylet to start. This is normally
+# fast, but when RAY_PREALLOCATE_PLASMA_MEMORY=1 is set, it may take some time
+# (a few GB/s) to populate all the pages on Raylet startup.
+RAYLET_START_WAIT_TIME_S = 120
```

Comment:
Why don't do `120 if env.get("RAY_PREALLOCATE_PLASMA_MEMORY") else 5`?

Reply:
It would be simpler to raise it for everyone (minimal downside), rather than adding such a special case.

Reply:
I see. I just thought it would be bad UX if someone needs to wait 120 seconds when there are actual issues (when GCS is actually not connected since it is common among new users). But we anyway writes warning, so I am okay with the current approach.